data/generatingScripts/replaceNoise.m

```diff
@@ -0,0 +1,13 @@
+function data = replaceNoise(data, amplitude, ~)
+%% generate noise (white gaussian) and replace the data with it
+% data = replaceNoise(data, power)
+% amplitude should be around 1 if low noise needed
+% TODO: should amplitude be a percentage? (of mean/std,...)
+
+    N = size(data,1);
+    noise = wgn(N,1,-25);
+    data(:,2) = noise*amplitude;
```

Comment:
new fn, the dataLoss needs new images (i cant do that), I've rebuilt the datasets already